### ISSUE TITLE
cannot deselect row in AG Grid in the top third of checkbox

### ISSUE THREAD

**Issue by varta5** [siemens/ix#1003]
### What happened?

Hello Team!

In **AG Grid** a selected row **cannot be deselected** by right-clicking on the **upper third part of the checkbox**.

In _dev tools_ I can see that the **input field is smaller** (16 x 9 px) **than** the _ag-checkbox-input-wrapper_ **div** or the _ag-checkbox_ **div** (both 16 x 16 px)

**Same** issue applies to the checkbox in the **header row** (if `columnDefs` contain `headerCheckboxSelection: true`)

Example code is the same as on in the iX docs:

https://ix.siemens.io/docs/grid/

![example_checkbox_ag_grid01](https://github.com/siemens/ix/assets/97350993/887983b5-87fc-4315-a8f4-393a72b70b49)

![example_checkbox_ag_grid02](https://github.com/siemens/ix/assets/97350993/9ec45e90-f8c6-4b8c-8611-7725a134de28)

![example_checkbox_ag_grid03](https://github.com/siemens/ix/assets/97350993/bf0bcdac-adcb-4a27-a1a6-c293bc0d4fb2)

### What type of frontend frameware are you seeing the problem on?

Angular

### Which version of iX do you use?

v2.0.4

### Code to produce this issue.

```shell
import { Component } from '@angular/core';
import { GridOptions } from 'ag-grid-community';

@Component({
  selector: 'app-example',
  template: `
    <ag-grid-angular
      style="height: 12rem; width: 100%"
      class="ag-theme-alpine-dark ag-theme-ix"
      [gridOptions]="gridOptions"
    ></ag-grid-angular>
  `,
})
export default class AGGrid {
  gridOptions = {
    columnDefs: [
      {
        field: 'type',
        headerName: 'Type',
        resizable: true,
        checkboxSelection: true,
      },
      {
        field: 'status',
        headerName: 'Status',
        resizable: true,
        sortable: true,
        filter: true,
      },
      { field: 'hwVersion', headerName: 'HW version', resizable: true },
    ],
    rowData: [
      {
        type: 'Equipment',
        status: 'Normal',
        hwVersion: '2.0',
        checked: false,
      },
      {
        type: 'Positioner',
        status: 'Maintenance',
        hwVersion: '1.0',
        checked: true,
      },
      {
        type: 'Pressure sensor',
        status: 'Unknown',
        hwVersion: 'N/A',
        checked: false,
      },
    ],
    rowSelection: 'multiple',
    suppressCellFocus: true,
    checkboxSelection: true,
  } as GridOptions;
}
```

**Comment by varta5** [comment]
Hello @nuke-ellington! When is the next step expected for this issue?

**Comment by varta5** [comment]
Hello @nuke-ellington and @robert-wettstaedt ! Does [Pull Request 1067](https://github.com/siemens/ix/pull/1067) resolve this Issue? Or is it for something else?

**Comment by nuke-ellington** [comment]
Hello @varta5, you are correct this should now be fixed (2.1.1). And sorry for the late reply!
